### ISSUE TITLE
Remove most, if not all usage of tt_wxString

### ISSUE THREAD

**Issue by KeyWorksRW** [KeyWorksRW/wxUiEditor#1075]
### Description:

TLDR; tt_wxString is not as valuable in wxWidgets 3.3 since 3.3 switches wxString to UTF8. If possible, switching all instances of this class to `tt_string` would be preferable since that class will seamlessly transition between a UTF16 or UTF8 version of wxString.

All of wxUiEditor's processing of strings is done assuming UTF8. That includes reading/writing to XML files (project, xrc, WinMerge directions, etc.), as well as all code generation. We generally use `tt_string` for storing and modifying strings. This is essentially `std::string` with a bunch of additional methods, including several methods for working with filenames in a cross-platform method.

Up until wxWidgets 3.3, wxString is a UTF16 string on Windows, so passing strings between `tt_string` and `wxString` requires a UTF conversion. We use wxString::utf8_string() to convert from, and wxString::FromUTF8() to convert to -- which is highly efficient with wxWidgets 3.3 where no UTF conversion is done.

Filenames are a special case since they need to be UTF16 to work on Windows. `tt_string` takes advantage of C++17 and uses `std::filesystem` for filename operations. `tt_wxString` uses `wxFileName` instead. Prior to wxWidgets 3.3, that saves a UTF8->UTF16 conversion, though the constructor for wxFileName far exceeds the cost of the UTF conversion that `tt_string` uses. With wxWidgets 3.3, there is not performance gain of starting with a UTF16 string, so the file manipulation functions like `tt_wxString::make_absolute()` are actually slower then the `tt_string version`.
